src/microbe_stage/Patch.cs

```diff
@@ -9,34 +11,36 @@
 [UseThriveSerializer]
 public class Patch
 {
-    /// <summary>
-    ///   List of all species and their populations in this patch
-    /// </summary>
-    [JsonProperty]
-    public readonly Dictionary<Species, long> SpeciesInPatch =
-        new Dictionary<Species, long>();
-
     [JsonProperty]
     public readonly int ID;
 
+    /// <summary>
+    ///   The current conditions of this patch.
+    /// </summary>
     [JsonProperty]
-    public readonly ISet<Patch> Adjacent = new HashSet<Patch>();
+    public readonly PatchConditions Conditions = new PatchConditions();
```

Comment:
I find this name a bit strange, as to me conditions is something that shouldn't include the species list.
Maybe call PatchConditions a PatchSnapshot if no other name is good. With those proxy properties no user of the Patch class (other than ones that need historical data), would need to know about that class.